packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -847,3 +939,9 @@ export interface Tag {
   readonly Key: string;
   readonly Value: string;
 }
+
+interface nestedStackTemplates {
+  updatedNestedStackTemplate: any,
```

Comment:
"Updated" seems like not a great word to use here. How about something like `cdkGeneratdTemplate` instead?